src/typings/global.d.ts

```diff
@@ -1,7 +1,9 @@
+import type { mailHandlerSingleton } from '@/server/notifications/email/mailHandler'
 import type { Prisma, PrismaClient } from '@prisma/client'
 import type { DefaultArgs } from '@prisma/client/runtime/library'
 
 declare global {
    // eslint-disable-next-line no-var
    var prisma: PrismaClient<Prisma.PrismaClientOptions, never, DefaultArgs>
+   var mailHandler: ReturnType<typeof mailHandlerSingleton>
```

Comment:
Above I have ignored the no-var rule. Either add it to this line too and remove the file from .eslintignore, or remove my no-var ignore, since now it is pointless.